RELEASE-NOTES.txt

```diff
@@ -3,7 +3,7 @@
 9.7
 ----
 - [*] In-Person Payments: Card Reader Manuals now appear based on country availability, consolidated into an unique view [https://github.com/woocommerce/woocommerce-ios/pull/7178]
------
+- [*] Login: Jetpack setup flow is now accessible from the Jetpack requirement screen. [https://github.com/woocommerce/woocommerce-ios/pull/7294]
```

Comment:
nit: maybe we can mention that this is available in the "login with store address" flow?

Reply:
Updated in 326e11b11dc6a4ba58d94aa8d1071e3c23099911 👍